components/footer.js

```diff
@@ -17,7 +17,7 @@ const nonprofitInfoText = `© Copyright ${new Date().getFullYear()}. The Teacher
 const Footer = (props) => (
   <div className='footer'>
     <hr />
-    <div className='footerLinks'>
+    <div className='footerLinks footer__child'>
       {links.map(({ key, href, label }) => (
         <div key={key}>
```

Comment:
since you edited .footerLinks > div as well, don't forget to add in the class to this div (line 22)

`div key={key} className='footerLinks__child'>`

Reply:
Thanks and done, anything else?